translations/bnpchs-final-2018-blusukan.html

```diff
@@ -0,0 +1,95 @@
+<!-- Title: Blusukan -->
+<h3>Description</h3>
+
+<p style="text-align: justify;">
+  Bollyjee presidential election is around the corner. Jelix as the presidential 
+  candidate wants to realize the national commitments by doing <i>blusukan</i> (blusukan 
+  is indonesian word that refer to hands-on interaction with public in order to notice their 
+  necessity).
+</p>
+
+<p style="text-align: justify;">
+  There are a bunch of islands and $N$ province in Bollyjee. The island of two provinces lie on could be the 
+  same or different. Jelix had $M$ information, each information provide two provinces 
```

Comment:
```suggestion
  same or different islands. Jelix has $M$ information, where each information provides two provinces 
```

kalau di Inggris, penggunaan "where" memang valid